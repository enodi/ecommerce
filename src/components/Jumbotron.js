import React from "react";

import FilterDropdown from "./FilterDropdown";

const Jumbotron = () => (
  <section className="jumbotron">
    <div>
      <h2>All Products</h2>
      <p>A 360Â° look at Lumin</p>
    </div>
    <FilterDropdown />
  </section>
);

export default Jumbotron;
