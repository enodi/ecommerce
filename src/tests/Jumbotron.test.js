import React from "react";
import { shallow } from "enzyme";

import Jumbotron from "../components/Jumbotron";

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Jumbotron />);
});

test("should render Jumbotron component correctly", () => {
  expect(wrapper).toMatchSnapshot();
});

test("should find text rendered on Jumbotron", () => {
  expect(wrapper.find("h2").text()).toEqual("All Products");
});
