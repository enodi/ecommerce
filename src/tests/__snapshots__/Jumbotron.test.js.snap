// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Jumbotron component correctly 1`] = `
<section
  className="jumbotron"
>
  <div>
    <h2>
      All Products
    </h2>
    <p>
      A 360Â° look at Lumin
    </p>
  </div>
  <FilterDropdown />
</section>
`;
